server.py

```diff
@@ -26,31 +26,19 @@ def index():
 def add_user():
     """Add a new user to user database with information provided. Send to user
        homepage via /login"""
-
-    first_name = request.form.get('first_name')
-    last_name = request.form.get('last_name')
-    email = request.form.get('email')
-    password = request.form.get('password')
-
-    QUERY = User.query.filter_by(email=email).first()
+    user_data = dict(request.form)
```

Comment:
This allows you to edit `user_data` because `request.form` is an ImmutableDict, which cannot be changed. This is more precautionary if you have data coming from a form that you need to remove.